antisymmetric_spec.rb

```diff
@@ -0,0 +1,29 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'matrix'
+
+describe "Matrix.antisymmetric?" do
+  it "returns true for an antisymmetric matrix" do
+    Matrix[[1, 2, 3], [4, 5, 6], [7, 8, 9]].antisymmetric?.should be_true
+  end
+
+  it "returns true for a 0x0 empty matrix" do
+    Matrix.empty.symmetric?should be_true
+  end
+
+  it "returns false for an asymmetric Matrix" do
```

Comment:
??? (An asymmetric matrix *can* be antisymmetric; what‘s the point of this test?)